app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -14,7 +14,7 @@ import AnalyticsContext from '../context/analytics';
  * Non-breaking space (`&nbsp;`) represented as unicode escape sequence, which React will more
  * happily tolerate than an HTML entity.
  */
-const NBSP_UNICODE = '\u00A0';
+export const NBSP_UNICODE = '\u00A0';
```

Comment:
I don't know that we ought to export the character for reuse, since it seems rather incidental for implementation and not something that one would expect as a named export of an encrypted upload module. I'd prefer if either (a) we duplicated it in `DocumentCaptureTroubleshootingOptions` or (b) moved it to some place where one _could_ reasonably expect a named export of the sort (e.g. `packages/document-capture/services/typography.ts`, `packages/typography/index.ts`).